src/main/java/com/photolocos/enterprise/dao/LocationDAO.java

```diff
@@ -55,7 +53,7 @@ public LocationDTO fetchByCoordinates(String longitude, String latitude) {
     }
 
     @Override
-    public LocationDTO fetchByArea(String state, String city) {
+    public LocationDTO fetchByStateAndCity(String state, String city) {
```

Comment:
You could just go with the name fetch, because the parameters give context about how it is being fetched.